### PR TITLE
Add support for Champion's Inspirational banner life regen

### PR DESCRIPTION
**Pull request by mthjones**
### Description of the problem being solved:

Champion's Inspirational node's life regen per banner stage mod was unsupported. This adds support using the existing config options.

### Steps taken to verify a working solution:
- Add a banner skill
- Allocate the Champion's "Inspirational" node
- Enable the config option "Is Banner Planted?"
- Confirm life regen is added and scaled with banner stages

### Link to a build that showcases this PR:
```
eNrNWlt32jgQfm5_hQ_PLWBuTXvI9pCEpOwJLQ1pu_vUI-wBtBGSa8uk9NfvSDbgUJyM1d092wdqzDe3T6PRjJ3-2-8r4a0hTriSpzW_3qx5IAMVcrk4rX26vXx5Unv72_P-hOnlh_lZyoX55bfnz_r22hOwBoFyNU-zeAH681ZT-ytqipjUS1ByzP5S8ZUKT2vvlYSaN2My5Hr7LRAsSd6zFZzWLlIQPNE1jyUByPB8_8v5kq0i1FzzVozLqQruQF_FKo2s8TWH-7EKETcaTz7c3BZMc1k0jZ4_608E20A81Ux7CX6c1gZIAFvABVvhJ2pjIkVVnbrfbZ90etlnq9Y4LjyNAMKdULP--r8DTmIYzucQaL6G85hrJEkG-wCaZXJVseNUaB4JDvEO79e7ZRLvflLuN0vV3yrNxMVkusO266_2pPvdf0dO6acD_8L18kzgSjhYMbKjheQaHIUniidK_kJ8RdHytVWrGZdOEW5FB-tFVdFBDOzDPEvcGxbyNNknykmZ0JhJdq4SXcipMug1n8MDaClyOKXhbrAs0JDGzQnEWLw0TcA4W0lgqvebsNUu9Ri-FYF-t3Q1LuA7TV8RWOrdSBZWqPOItiKwWe7bWmlb9Ck07lA9_5HKGrwx2JEMaEo_yRgSiNeFglyu_qFAvqyUKmgEb2ABsrCXmt3H0NcAwfIKj8IbpoGWl_uD7dWj_BgsiR8DPMJPufqHAhX4MYIP-WnWHy0VFen5wuKQUC4kxIvNdMlBENCWy6IIidOiwEHEFJGKcQ_XLCnur_L6m0WTwYuBlJf5MeDhiRIhUFuN6hKTWP1l2h9RTWwQr1QaE5cwA9Niniw3CQ_wALad5A2EaUCrX7s2bqzWsMJdYXtAbGn3RstEzwQ2w9TIUa0QlSQGWrPg7kKFC6hkpLrENI0irA0mw6hylzxGlhNeODhf9gjoD9jwn7OI0iGZbUg1sEeTDVzzxVJLnKnoVg5E6LEsmUoqBLOHk01cpkKQOs_pHX-I1Gwm4I3X_O5D4L_qdsu3vLpHZ5Zm2kwIBaKAHrN9A1Pa6FzGIH9syPofwEkGhjJMY5PdZBuHEj-b6TfsNG6uRqtIxdrePGciSKzKkYxS7Uk7SK94EnydpfO5mZdraCK2o_7w8nJ4fjv6PMy9KIokZrG-ynQ1MwNg9v--IE3B1jgvUEKwKAEctedMJKibh_lCT7GwB5qEx-ExH8MpaDy3g5gjiRSwmfpILtvxm4I0szEJiKvCBNAJu91EYDIgoXKWF0wK2s60JGQ2hNICtMMubRkgYBsSctfMkNBDAQMuzNlJY22MGyE7dml4PAhjPks1MdvsRELyw7TmpACL7Sdxd9CAeT9EciLvGCnY_KyhJSW2JORlu4A5YLoTt70tJ7fsDmReLLeFsW_3WuIlWDGvYJWcbfBEujT0HjwaeQDAsypOUXcIc5YKc_9jygTXG1PCC3evs0ejLbyZLNW96W4yNWZzJ0j69XX2y0DoXIOxsfU9i8w4aJ94Dmx3aL9b9-1jTy4DkYY4zObn7i5uwWbGds1bmEek5yqVeI5ILszDXXPQhtsgDlQbrcbwsz66kmOvhJox0crlxT6q4s_-VqE9K0YmUVisl99SpL3mfcvCG9ndb_nJgRME2UgWsDK_jkGzkGnWGGmkomH4aFjH8OqIQsv4QUQ7vXuagix-vDLnmWnqH7iXJVFm5__H-lY3lXfcSSrExsEk7xmT0hzZx_gnMI7140DDMcKPsYuSXi56wG5-ZbffbQzgsYwHK-bnWY9fim8AjJ9tw1qiId7kNdvwLLGFwYtu8_VJ70Wr8_qk86LV9X3_he93_FbN02ig8EbC7-UvG4zCTk75MMSDK3xvNDWyO59uru3Fs6XWUfKm0bi_v69HTC_VHL7jqFkP1KoRoRr0-6Xl_aUx1Bjgv7PFx4vBavTH9e8f8dvpqVXY2GrsZ-8tki0nGKelxBBhLt4rPF3Mb-bm9oul6TOHey8BbHeWUx2b2H8otfoTG3jfb3XqrzpZKXkHTI9ZtMsaA8rrUC8vQ9hVXHCkMLYVdLuIBviHGQdOOvVOL3tw17cpkS-QuZ5Cts5pAlhylQy_AIuUtLcLRSuHPgKzdXmb81Ohto1mBvT86T2LvMFskyRMeBlnXheldJazead8VPRQrEUTexx1ppROHB06FssTTp2B0BVFLjGt77w2yaXWMZfabvTSSGj9Q2IdNyd7btZ67my65NMTK_AOsLfVVQ0dyaSei50n0g8Po7UZZasrbrvvIt_FXsctF3y3zOs8vTwUYquqPZZejoGT6k7XwR-HGth-yky48bJRpuLyHWfZKb-6LkKkLUlZCKc4u860thzWvee2k9pu6duunmUd97LvkNO-M_ltSkJ0KCAHt3-h-SGupPsq9Byy0n0ZfiFdqCXZyBJ3DR3qwNwNlwtaJ1C5UJTS61gu6OvpEk6bwJPvvgwdZ7a6LtFQ9ot7kj_h0mCViiNnW7-RT5F2PraDqH2loeScL356QTGzjxpwtLXvU7YvKHbvkH6GTgST2vzJwkwpAUzmY3D-eDA30m8c_tXj37AF3a8=
```

### Before screenshot:
image

### After screenshot:
image
image

### CODE REVIEW

**Comment by mthjones** [comment]
Note that this causes the life regen to stick around if the config is not disabled before disabling the banner skill...investigating.

File: src/Data/Skills/act_str.lua [hunk 1]
```diff
@@ -7590,6 +7592,7 @@ skills["BloodstainedBanner"] = {
 	baseMods = {
 		skill("radius", 46),
 		skill("manaReservationPercent", 0, { type = "Condition", var = "BannerPlanted" }),
+		flag("Condition:AffectedByPlacedBanner", { type = "Condition", var = "BannerPlanted" }, { type = "GlobalEffect", effectType = "Buff" })
```
Comment (LocalIdentity) [inline src/Data/Skills/act_str.lua:7595]: Need to remember to add these lines to the act_str.txt file so they are exported from the game files. I'll add them this time

**Review by LocalIdentity — APPROVED** [review]

File: src/Data/Skills/act_str.lua (hunk 1 above)
Comment (mthjones) [reply src/Data/Skills/act_str.lua]: Ah. Probably missed this in the Nightblade fix then as well. 🙈 

I'll fix that tomorrow.

Comment (mthjones) [reply src/Data/Skills/act_str.lua]: nvm, already fixed in [this commit](https://github.com/PathOfBuildingCommunity/PathOfBuilding/commit/591430285843171b7bece76723204d395cae5bd4#diff-c4d2f4bc4ef4a1fd2a1991410919c41890defce4960634338eaa8f4cc370c556R336). 👍

**Pull request merged**